### ISSUE TITLE
You can set different hotkeys to the same key

### ISSUE THREAD

**Issue by Skarletto** [space-wizards/space-station-14#21261]
## Description

fun fun throw 5km/h
**Reproduction**

Set a hotkey to a key that's in use
**Screenshots**

https://github.com/space-wizards/space-station-14/assets/122584947/1708e3ac-e167-4839-8d08-f564043e612e

**Additional context**

**Comment by CatMagic1** [comment]
haha macros

**Comment by QuietlyWhisper** [comment]
I don't think being able to bind stuff to the same key is an issue more than being able to do this, which should have a cooldown between throws anyway.

That being said I don't know many use cases where you'd NEED to have the same key bound twice, but hey, freedom of choice I guess?

**Comment by Skarletto** [comment]
Administratively it's hard to tell whether someone's doing THIS or using legit macros which sucks imo, and we often get reports of people 'cheating' because they're throwing bolas at lightspeed. Idk it's weird to me that you can set a hotkey and automatically just be better at stunning people. Keep in mind this literally applies to anything that has an interaction on-hit (I know bolas stun with 2 only but whatever)

https://github.com/space-wizards/space-station-14/assets/122584947/208ca2eb-187b-4424-b626-22a23bfd758f

**Comment by TemporalOroboros** [comment]
Probably just needs a throw cooldown.[^1]


[^1]: This message by someone that currently has action2 and equip-from-back bound to the same key to speed up readying a gun.

**Comment by deathride58** [comment]
Being able to bind multiple actions to the same key ingame is not a design flaw at all; if a bad-faith individual wishes to have multiple actions on the same key, such as the example of taking an item from the backpack and throwing it on the same frame, then removing the ability to set that up ingame genuinely won't stop them from being able to do that (as it's straight-forward to set up an AHK macro or similar to do the same thing and end up right back to square one)

The ability to assign multiple actions to the same key is genuinely useful for accessibility, and absolutely should stay; it'd be a massive blow to the game's accessibility if it were to be removed. In our own personal case, we have pull object and move pulled object bound to the same singular mouse button, as the default bindings are particularly rough on our wrist, and we've found that it's far more convenient for us to have both of those actions on a button that doesn't cause our arthritis to flare up if we need to spam it (whether it's for fighting with someone trying to pull away something we're pulling, or if we're trying to nudge something into a specific spot). It's strongly likely there's other players in a similar boat of relying on this behavior to accommodate for physical conditions like ours.

For the case of players being able to take an item out of their backpack and throw it on the same frame, it'd be far more reasonable to instead focus on the cause, rather than the symptom: there's a complete lack of cooldown period between an item being placed in your hand, and that item being usable. A cooldown as marginal as .1 seconds would resolve the actual issue at hand whilst mostly reducing the impact of out of game macros (a tenth of a second is more than enough to make a macro mostly pointless), and not affecting actual good-faith players much at all

**Comment by Kayek14** [comment]
Ive seen people do this with stun batons now since the bolas were nerfed. Throwing three batons and stunning someone in 0.5 seconds is very strong.

**Comment by QuietlyWhisper** [comment]
> Ive seen people do this with stun batons now since the bolas were nerfed. Throwing three batons and stunning someone in 0.5 seconds is very strong.

Yeah I've been saying throwing is still a major issue, even if you bandaid stun baton theres always an edge case; you can spam throw glass and d4 at diona to kill them lmao
